lib/src/lib.rs

```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![deny(unused_must_use)]
 //! Jujutsu version control system.
 
+#![deny(missing_docs, unused_must_use)]
```

Comment:
Maybe `warn(missing_docs)` is better for local development? I tend to leave draft patches without docs. I think CI should deny it, though.

Reply:
@yuja do you think we should also adjust `unused_must_use` to `warn`?